org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/workflowEngine/Action.java

```diff
@@ -0,0 +1,18 @@
+package org.eclipse.ice.modeling.workflowEngine;
+
+public class Action {
```

Comment:
As we discussed, we should work to interface this with the Commands package since Commands will do most of what we need Action to do

Reply:
Agree

Reply:
Definitely agree on this part, but the translation is not 1-1. For example, what if it is a native function execution? Then is it really a call to Commands? (I don't know the answer - just saying we should think about it.)